backend/googlecloudstorage/googlecloudstorage.go

```diff
@@ -248,6 +248,18 @@ Docs: https://cloud.google.com/storage/docs/bucket-policy-only
 				Value: "DURABLE_REDUCED_AVAILABILITY",
 				Help:  "Durable reduced availability storage class",
 			}},
+		}, {
+			Name:     "directory_markers",
+			Default:  false,
+			Advanced: true,
+			Help: `Upload empty object with a trailing slash in name when new directory is created
+
+Empty folders are unsupported for bucket based remotes, so we create an empty 
+object named "/", to persist this folder. The directory marker object 
+is filtered from file listing.
+
+See: https://github.com/rclone/rclone/issues/3453
```

Comment:
I don't think we need a ref to the issue here

Reply:
Changed; Done